### PR TITLE
created json files, created queries, started pug page, created schema

### PR DESCRIPTION
**Pull request by sashajustice**

### CODE REVIEW

File: database/database.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const connectionString = "postgres://SashaJustice@localhost:5432/Platypus"//connects db to ur machine
+const pgp=require('pg-promise') () //importing pg-promise into this file
+const db = pgp( connectionString) //invoking pg-promise onto db
+
+const getTask=() =>
```
Comment (shakalee14) [inline database/database.js]: add a space after function name and before parentheses for readability 

``` const getTask = () => ```

File: database/database.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const connectionString = "postgres://SashaJustice@localhost:5432/Platypus"//connects db to ur machine
```
Comment (shakalee14) [inline database/database.js]: do you want to keep these comments in?

File: database/schema.sql [hunk 3]
```diff
@@ -0,0 +1,7 @@
+DROP TABLE IF EXISTS todoitem;
+
+CREATE TABLE IF NOT EXISTS todoitem(
+  ID SERIAL PRIMARY KEY,
+  task VARCHAR(20) NOT NULL,
```
Comment (shakalee14) [inline database/schema.sql]: 20 characters for a task seems small but you don't have to change it

**Comment by kousagi1012** [comment]
Do we have to manually delete the folder? whenever we push it, it says it's up to date, but the node_modules are still there

**Pull request merged**